docs/src/components/home/sections/FigmaSection.tsx

```diff
@@ -0,0 +1,146 @@
+  Icon,
+  Rating,
+  Text,
+  View,
+} from '@aws-amplify/ui-react';
+
+import { CodeHighlight } from '@/components/CodeHighlight';
+import { DataIcon } from '@/components/Icons';
+import { HomeCTA } from '@/components/home/HomeCTA';
+import { useIntersectionObserver } from '@/components/useIntersection';
+import { FigmaLogoMono } from '@/components/Logo';
+
+const StudioCard = () => {
+  return (
+    <View className="docs-home-figma__node">
+      <View className="docs-home-figma__node-label">
+        <Icon
+          ariaLabel=""
+          viewBox={{
+            width: 12,
+            height: 12,
+          }}
+          pathData="M3.743 2.748L6 .5l2.257 2.248L6 4.996 3.743 2.748zm-.995 5.51L.5 6l2.248-2.257L4.996 6 2.748 8.257zm5.51.994L6 11.5 3.743 9.252 6 7.004l2.257 2.248zM11.5 6L9.252 3.743 7.004 6l2.248 2.257L11.5 6z"
+        />
+        ProductCard
+      </View>
+      <View className="docs-home-figma__node-handles" />
+      <View className="docs-home-figma__node-handles" />
+      <Card variation="elevated">
+        <Flex direction="column">
+          <Text
+            className="docs-home-figma__data"
+            fontSize="large"
+            fontWeight="bold"
+          >
+            AWS Amplify
+          </Text>
+          <Rating className="docs-home-figma__data" value={3.5} />
+          <Text
+            className="docs-home-figma__data"
+            fontSize="xl"
+            color="font.tertiary"
+          >
+            $99
+          </Text>
+          <Button variation="primary">Add to cart</Button>
+        </Flex>
+      </Card>
+    </View>
+  );
+};
+
+export const FigmaSection = () => {
+  const ref = useRef<HTMLDivElement | null>(null);
+  const entry = useIntersectionObserver(ref, {
+    threshold: 0.25,
+    freezeOnceVisible: true,
+  });
+  const isVisible = !!entry?.isIntersecting;
+
+  return (
+    <View
+      as="section"
+      id="figma"
+      className={classNames(
+        'docs-home-section',
+        'docs-burst-bg',
+        'fade-in',
+        isVisible && 'shown'
+      )}
+      ref={ref}
+    >
+      <Flex className="docs-home-subsection" direction="column" gap="large">
+        <Heading level={2}>
+          Build UI <strong>visually</strong> in Figma
+        </Heading>
+        <Flex
+          direction={{
+            base: 'column',
+            large: 'row',
+          }}
+          gap="large"
+        >
+          <Flex gap="large" direction="column" alignItems="flex-start" flex="1">
+            <Text className="docs-home-text">
+              With Amplify Studio you can design components in Figma, bind them
+              to your data, and generate production-ready React code. Amplify
+              Studio's UI builder makes it easy to build UI that is connected to
+              your data that is customizable and configurable in your codebase.
+              Go from design to production-ready code in minutes and eliminate
+              the design-development gap.
```

Comment:
Just a suggestion: "With Amplify Studio, you can design components in Figma, bind them to your data, and generate production-ready React code. ~~Amplify Studio's UI builder makes it easy to build UI that is connected to your data that is customizable and configurable in your codebase.~~ Go from design to production-ready code in minutes and eliminate the design-development gap."